README.md

```diff
@@ -1,19 +1,19 @@
-# bs-react-intl-ppx
+# melange-react-intl-ppx
+
+These are the [Melange](https://melange.re/) bindings for [react-intl](https://github.com/formatjs/formatjs/tree/main/packages/react-intl).
 
 ## Installation
-Get the package:
+Install [opam](https://opam.ocaml.org/) package manager.
+Then:
 
 ```shell
-# yarn
-yarn add --dev @ahrefs/bs-react-intl-ppx
-# or npm
-npm install --save-dev @ahrefs/bs-react-intl-ppx
+opam pin add melange-react-intl-ppx.dev git+https://github.com/ahrefs/bs-react-intl-ppx.git#eb36bab7565552fffcc59917913220f52c643db8
```

Comment:
```suggestion
opam pin add melange-react-intl-ppx.dev git+https://github.com/ahrefs/melange-react-intl-ppx.git#master
```
Should update this line after PR is merged and repo is renamed

Reply:
https://github.com/ahrefs/bs-react-intl-ppx/blob/9e7418251f817c0a44d47820f923285702de54dd/README.md?plain=1#L10

Updated in advance.